lib/tasks/test_track_rails_client_tasks.rake

```diff
@@ -31,3 +39,4 @@ end
 task 'db:schema:load' => ['test_track:schema:load']
 task 'db:structure:load' => ['test_track:schema:load']
 task 'db:migrate' => ['test_track:migrate']
+task 'assets:precompile' => ['test_track:generate_build_timestamp']
```

Comment:
going to try reordering the tasks and adding a flag to not run the initializers until this is done here